viewer/add-heading-anchors.js

```diff
@@ -101,5 +91,14 @@ var addHeadingAnchors = {
       this.handler.on("error", clipboardErrorHandler);
       this.handler.on("success", ensureSuccessHandler);
     }
+  },
+
+  registerDismissPopoverHandler: function () {
+    $(this.target).click(function (e) {
+      // if clicking a non-popover-ed element
```

Comment:
the below conditional makes it so that it dismisses the popover only if the thing you're clicking is NOT inside an 'a.headerLink'.  There are some headaches due to the way clipboardJS binds click events + strictly checking e.target is insufficient, since the click event will first fire on the `<i>` tag for the fontawesome icon